src/levenshtein.rs

```diff
@@ -88,10 +88,30 @@ pub const RDAMERAU_COSTS: EditCosts = EditCosts{mismatch_cost: 1, gap_cost: 1, s
 ///
 /// assert!(dist == 1);
 /// ```
-pub fn levenshtein_naive(a: &[u8], b: &[u8]) -> u32 {
+pub fn levenshtein_naive<T: PartialEq>(a: &[T], b: &[T]) -> u32 {
     levenshtein_naive_with_opts(a, b, false, LEVENSHTEIN_COSTS).0
 }
 
+/// Returns the Levenshtein distance between two strings using the naive scalar algorithm.
+///
+/// # Arguments
+/// * `a` - first string (&str)
+/// * `b` - second string (&str)
+///
+/// # Example
+/// ```
+/// # use triple_accel::*;
+/// # use triple_accel::levenshtein::*;
+/// let dist = levenstein_native_str("abc", "ab");
+///
+/// assert!(dist == 1);
+/// ```
+pub fn levenstein_native_str(a: &str, b: &str) -> u32 {
```

Comment:
This should be changed from `native` to `naive` right?

Reply:
yep!